README.md

```diff
@@ -137,7 +137,26 @@ myConfig.listener = (emitter) => {
 });
 ```
 
-## electrode-confippet
+### logLevel
+
+You can control how much output the Electrode Server logs to the console by setting the `logLevel` property in the config to "info" (the default if this is not specified at all), "warn" or "error". A level of "warn" means only warning and error messages will be printed.
+
+```js
+{
+  electrode: {
+    suppressStartupBanner: true
```

Comment:
`suppressStartupBanner` should be `logLevel?

Reply:
Whoops.  Yes, seems like it should.  Thanks.